packages/notion-types/src/collection.ts

```diff
@@ -39,6 +39,7 @@ export interface Collection {
   copied_from: string
   template_pages?: Array<ID>
 
+  cover?: string;
```

Comment:
Looks like you are just adding the cover type. More work probably needs to be done to add this in code.